controllers/topology/desired_state.go

```diff
@@ -143,6 +150,8 @@ func computeControlPlane(_ context.Context, s *scope.Scope, infrastructureMachin
 		cluster:               cluster,
 		namePrefix:            fmt.Sprintf("%s-", cluster.Name),
 		currentObjectRef:      currentRef,
+		// Note: It is not possible to add an ownerRef to Cluster at this stage, otherwise the provisioning
```

Comment:
is this referring to e.g https://github.com/kubernetes-sigs/cluster-api/blob/19653d96df3bc151aa02cd837814cafd517ee9dc/controlplane/kubeadm/controllers/controller.go#L134-L137
Do we have a doc we can point at where this is documented as a contract?

Reply:
same as https://github.com/kubernetes-sigs/cluster-api/pull/5523#discussion_r744968417